app/src/main/java/com/example/coopertest/SettingsActivity.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.coopertest
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.edit
+//import androidx.preference.PreferenceFragmentCompat
+import com.example.coopertest.PrefsFragment as PreferenceFragmentCompat
+
+class SettingsActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.settings_activity)
+        supportFragmentManager
+            .beginTransaction()
+            .replace(R.id.settings, SettingsFragment())
+            .commit()
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
+    }
+
+    class SettingsFragment : PreferenceFragmentCompat() {
+        override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
+            setPreferencesFromResource(R.xml.root_preferences, rootKey)
```

Comment:
Same as above, unresolver reference error in IDE but the app works :laughing: so it can probably stay this way if it's needed

Reply:
No problem in my IDE too